### DISCUSSION TITLE
How does the Subclipse help contribution work? Why isn't it part of the build?

### DISCUSSION THREAD

**Discussion by IliyanLaskov** [subclipse/subclipse#290] [Q&A] [answered]
As far as I can see, a plugin for an Eclipse help contribution is present 'subclipse.doc' but it is not a part of the Maven build and isn't packaged in the repository. Why is that? Are the help texts just old?

**Comment by markphip** [comment, marked as answer]
I do not recall if there was a reason. The text is definitely old, probably 15 years at least.

Subclipse itself really is not maintained any more either. I do my best if someone submits a PR but no one has worked on it in maybe 6+ years
